CASMINST-5307 (#2761)
Cray-HPE/docs-csm: operations/utility_storage/Troubleshoot_Ceph_Services_Not_Starting.md

   At this point, the processes are starting/running on the node that crashed; this may take a few minutes.

>    If after five mins the services are still reporting down, then fail-over the Ceph mgr daemon and re-check the daemons:

Reviewer: ```suggestion
   If after five minutes the services are still reporting down, then fail-over the Ceph mgr daemon and re-check the daemons:
```

   ```bash
   ceph mgr fail $(ceph mgr dump | jq -r .active_name)
   ```

## Services not starting due to `ssh` keys not cached on nodes

```bash
num_storage_nodes=$(ceph node ls|jq -r '.osd|keys|length')
ceph cephadm get-pub-key > /etc/ceph/ceph.pub
for node in $(seq 1 $num_storage_nodes); do
   nodename=$(printf "ncn-s%03d" $node)
   ssh-keyscan -t rsa -H $nodename >> ~/.ssh/known_hosts
done
```

> You may also want to clear the local cache entries in the `known_host` files on the nodes.

Reviewer: ```suggestion
It is recommended to also clear the local cache entries in the `known_host` files on the nodes.
```

```bash
for node in $(seq 1 "$num_storage_nodes"); do

>  nodename=$(printf "ncn-s%03d" "$node")

Reviewer 2: Not a problem but the indentations are inconsistent.
